dev/breeze/doc/08_ci_tasks.rst

```diff
@@ -114,23 +114,6 @@ These are all available flags of ``get-workflow-info`` command:
   :width: 100%
   :alt: Breeze ci get-workflow-info
 
-Finding backtracking candidates
```

Comment:
Very nice to get rid of this section :)

Reply:
Not needed any more

Reply:
Yeah, exactly, it's lovely that it's not needed! :smile: